iep-module-dynconfig/src/test/java/com/netflix/iep/dynconfig/DynamicConfigServiceTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2014-2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.iep.dynconfig;
+
+import com.netflix.iep.config.ConfigManager;
+import com.netflix.iep.config.DynamicConfigManager;
+import com.netflix.spectator.api.NoopRegistry;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.Properties;
+
+public class DynamicConfigServiceTest {
+
+  private final DynamicConfigManager manager = ConfigManager.dynamicConfigManager();
+
+  private DynamicConfigService newService() {
+    return new DynamicConfigService(new NoopRegistry());
+  }
+
+  @Before
+  public void before() {
+    manager.setOverrideConfig(ConfigFactory.empty());
+  }
+
+  @Test
+  public void fastProperties() {
+    Properties props = new Properties();
+    props.setProperty("iep.test.which-config", "dynamic");
+    DynamicConfigService service = newService();
+    service.updateDynamicConfig(props);
+    Assert.assertEquals("dynamic", manager.get().getString("iep.test.which-config"));
+  }
+
+  @Test
+  public void override() {
```

Comment:
Covers the override behavior nicely.

Maybe consider adding assertions for the configs brought in by the ConfigManager `netflix.iep.include` functionality?

Added these tests locally, which pass:

```
Assert.assertEquals("abc", manager.get().getString("includes.a"));
Assert.assertEquals("def:foo", manager.get().getString("includes.b"));
```

Maybe a bit belt-and-suspenders, but it makes the base case expected behavior clear for future readers.